sentry/src/test/java/io/sentry/GsonSerializerTest.kt

```diff
@@ -152,6 +152,7 @@ class GsonSerializerTest {
 
     @Test
     fun `when deserializing unknown properties, it should be added to unknown field`() {
+        // TODO: Failure irrelevant after JsonSerializer branch is merged.
```

Comment:
should we address this before merging?

Reply:
yeah, good idea. Best to wait until #1744 is merged.